docs/src/route.ts

```diff
@@ -35,30 +35,30 @@ const ROUTES = {
     ? '/material/getting-started/support/#community-help-free'
     : '/getting-started/support/#community-help-free',
   blog: FEATURE_TOGGLE.enable_blog_index ? '/blog/' : 'https://medium.com/material-ui',
-  showcase: FEATURE_TOGGLE.enable_blog_index
+  showcase: FEATURE_TOGGLE.enable_redirects
```

Comment:
My bad. used the wrong feature toggle 🥲

Reply:
Would removing the legacy feature toggles help 😁?